### PR TITLE
Create holding page 

### PR DESCRIPTION
**Pull request by MartinBenediktBusch**
### This PR addresses the following tasks:

1. Designed and created a holding page to acknowledge proposal submissions and provide essential information about the review process.
2. Configured the system to redirect users to the holding page upon successful proposal submission.
3. Implemented a link on the holding page that redirects users to the event registration page, enabling them to edit their submission if needed.

Additionally, I've added a feature to ensure that the registration page gets reloaded upon redirection. This ensures that the data is always up to date, providing users with the latest state of their submission.

### CODE REVIEW

File: packages/berlin/src/pages/Holding.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    queryFn: fetchEvents,
+  });
+
+  if (eventsLoading) {
+    return <Body>Loading events...</Body>;
+  }
+
+  // Define onClick handler to navigate to registration page and trigger reload
+  const handleRegistrationClick = () => {
+    navigate(`/events/${events?.[0]?.id}/register`, { replace: true });
```
Comment (camilovegag) [inline packages/berlin/src/pages/Holding.tsx:27]: We can get the `eventId` from the url by using react-router `useParams` hook, that would remove the need of doing the `events` query here. Also, the `replace` replaces the history navigation, so user won't be able to go back, not sure if thats what we want, as you'll face this scenario:

https://github.com/lexicongovernance/forum-frontend/assets/59750365/e9862619-27b4-401f-be77-cc5c57a04da8

File: packages/berlin/src/pages/Holding.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  });
+
+  if (eventsLoading) {
+    return <Body>Loading events...</Body>;
+  }
+
+  // Define onClick handler to navigate to registration page and trigger reload
+  const handleRegistrationClick = () => {
+    navigate(`/events/${events?.[0]?.id}/register`, { replace: true }); 
+    window.location.reload();
```
Comment (camilovegag) [inline packages/berlin/src/pages/Holding.tsx:28]: In theory, react query handles this when we invalidate the queries, but as you can see in the posted video this is not working? My registration fields didn't updated. doing a window.location.reload() is a bit agressive as it "messes" with our single page application user experience where everything is smooth, lets check what happened here.

cc @diegoalzate

**Review by camilovegag — CHANGES_REQUESTED** [review]

File: packages/berlin/src/pages/Holding.tsx (hunk 2 above)
Comment (diegoalzate) [reply packages/berlin/src/pages/Holding.tsx:28]: I agree, a hard reload might cause an unexpected state. If it does not work without the reload then we should think about a different approach

File: packages/berlin/src/pages/Holding.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  };
+
+  return (
+    <FlexColumn $gap="2rem">
+      <Body>Thank you for submitting!</Body>
```
[the remark this answers is not in the document]
Comment (MartinBenediktBusch) [reply packages/berlin/src/pages/Holding.tsx]: Implemented title component.

File: packages/berlin/src/pages/Holding.tsx (hunk 1 above)
Comment (MartinBenediktBusch) [reply packages/berlin/src/pages/Holding.tsx]: Done.

**Review by camilovegag — APPROVED** [review]

File: packages/berlin/src/pages/Holding.tsx (hunk 2 above)
Comment (MartinBenediktBusch) [reply packages/berlin/src/pages/Holding.tsx]: Issue: https://github.com/lexicongovernance/forum-frontend/issues/146

**Pull request merged**